datastore/src/main/java/com/ds/datastore/Utilities.java

```diff
@@ -44,7 +46,7 @@
                 .build()
                 .send(request, HttpResponse.BodyHandlers.ofString());
         if(requestType.equals("POST") && (response.statusCode() != 201 && response.statusCode() != 200) ){
-                System.out.println(response.statusCode());
+                logger.warn("Did not receive a successful status code");
                 throw new RuntimeException();
```

Comment:
and I still want a more helpful message here :)

Reply:
Changed it to : {} received, POST failed", response.statusCode()

Reply:
I meant in the `RuntimeException`

Reply:
We only threw that to force a retry. Didn't really intend it to be descriptive of what actually happened

Reply:
oh I see.